src/payload.jl

```diff
@@ -25,6 +25,7 @@ function insertsorted!(arr::Vector{T}, x::T, by=identity) where {T}
 end
 
 # Like a Channel, but when you take! a Payload from it, it will be the next one in order
+# take! is not threadsafe
 mutable struct PayloadOrderer{B, C<:AbstractParsingContext} <: AbstractChannel{ParsedPayload{B,C}}
```

Comment:
whenever i aask about parallel csv loading, i always want to know how order is guaranteed. maybe at least worth mentioned in the README that ChunkedBase takes care of that?

Reply:
I wasn't sure what is the best place to mention it... maybe the `consume!` method docstring?